ansible/tasks/setup-supabase-internal.yml

```diff
@@ -6,7 +6,7 @@
 
 - name: AWS CLI
   get_url:
-    url: "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-{{ aws_cli_release }}.zip"
+    url: "https://awscli.amazonaws.com/awscli-exe-linux-aarch64-{{ aws_cli_release }}.zip"
```

Comment:
This should be dynamically configured [like this](https://github.com/supabase/postgres/blob/d26bec8d840e363f21fa9fe3c1acbe33f2363787/ansible/tasks/setup-misc.yml#L16) right?

Reply:
done